src/v/archival/probe.cc

```diff
@@ -113,6 +113,13 @@ void ntp_level_probe::setup_public_metrics(const model::ntp& ntp) {
            "Total number of accounted segments in the cloud for the topic"),
          labels)
          .aggregate(aggregate_labels),
+       sm::make_current_bytes(
+         "segments_metadata_bytes",
+         [this] { return _segments_metadata_bytes; },
+         sm::description("Current number of bytes consumed by remote segments "
```

Comment:
Description says "by the partition" but I think this is actually aggregated by topic in the output? (via aggregate_labels).  I'm good with having it aggregated by topic (per-partition metrics are expensive), just needs the description to line up.

Reply:
I ended up moving this into the partition probe, which is aggregated by partition. I am also wary of metric cardinality problems, so I can find it another home if there's concern that it doesn't belong.